iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -955,4 +1046,44 @@ private void validateBasicStats(Table icebergTable, String dbName, String tableN
       Assert.assertEquals(summary.get(entry.getValue()), hmsParams.get(entry.getKey()));
     }
   }
+
+  private void validateMigration(String tableName, int recordCount) {
+    List<Object[]> originalResult = shell.executeStatement("SELECT * FROM " + tableName + " ORDER BY a");
```

Comment:
Can we minimize the time spent here?
* Do we need this query, or we can expect that the writer of the test knows the expected number?
* Do we need it to be ordered?